community/user-stories/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: single
+title:  "Spinnaker User Stories"
+sidebar:
+  nav: community
+redirect_from: /community/user-stories
+---
+
+Nothing speaks more powerfully than actual user stories from people evaluating and using Spinnaker in production. If you have your own User Story to share just [email us]( mailto:user-story@Armory.io?subject=Spinnaker%20User%20Story) and we'll schedule one with you in person or via video conference and post it here.
```

Comment:
s/us/Armory/

...just because I think users might get confused by a page referring to an "us" that is Armory instead of the whole Spinnaker OSS community.

Reply:
I'll update that